ambari-server/src/main/java/org/apache/ambari/server/controller/internal/ClusterStackVersionResourceProvider.java

```diff
@@ -625,6 +627,8 @@ RequestStageContainer createOrchestration(Cluster cluster, StackId stackId,
       if (RepositoryType.STANDARD != repoVersionEnt.getType()) {
         ClusterVersionSummary clusterSummary = desiredVersionDefinition.getClusterSummary(cluster);
         serviceNames.addAll(clusterSummary.getAvailableServiceNames());
+      } else {
+        serviceNames.addAll(ami.getStack(stackId).getServices().stream().map(ServiceInfo::getName).collect(Collectors.toList()));
```

Comment:
how about breaking this line on multiple lines?

Reply:
I made it shorter by pushing it into StackInfo.